lib/draper/automatic_delegation.rb

```diff
@@ -6,8 +6,7 @@ module AutomaticDelegation
     def method_missing(method, *args, &block)
       return super unless delegatable?(method)
 
-      self.singleton_class.delegate method
-      send(method, *args, &block)
+      object.send(method, *args, &block)
```

Comment:
Sounds good. I added documentation to the source and README. I also added some additional tests to cover the scenarios. I'm going to go ahead and merge this then.